### PR TITLE
Update logging.md

### PR DESCRIPTION
**Pull request by nwanati**
Updated some sentences for clarity.

### CODE REVIEW

File: content/en/docs/concepts/cluster-administration/logging.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ weight: 60
 Application logs can help you understand what is happening inside your application. The logs are particularly useful for debugging problems and monitoring cluster activity. Most modern applications have some kind of logging mechanism. Likewise, container engines are designed to support logging. The easiest and most adopted logging method for containerized applications is writing to standard output and standard error streams.
 
 However, the native functionality provided by a container engine or runtime is usually not enough for a complete logging solution.
-For example, you may want access your application's logs if a container crashes; a pod gets evicted; or a node dies.
+For example, you may want access to your application's logs if a container crashes; a pod gets evicted; or a node dies.
```
Comment (sftim) [inline content/en/docs/concepts/cluster-administration/logging.md:15]: I'd write:
```suggestion
For example, you may want to access your application's logs if a container crashes; a pod gets evicted; or a node dies.
```

**Review by sftim — COMMENTED** [review]
/lgtm

File: content/en/docs/concepts/cluster-administration/logging.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ weight: 60
 Application logs can help you understand what is happening inside your application. The logs are particularly useful for debugging problems and monitoring cluster activity. Most modern applications have some kind of logging mechanism. Likewise, container engines are designed to support logging. The easiest and most adopted logging method for containerized applications is writing to standard output and standard error streams.
 
 However, the native functionality provided by a container engine or runtime is usually not enough for a complete logging solution.
-For example, you may want access your application's logs if a container crashes; a pod gets evicted; or a node dies.
+For example, you may want to access your application's logs if a container crashes; a pod gets evicted; or a node dies.
```
Comment (wiggitywhitney) [inline content/en/docs/concepts/cluster-administration/logging.md:15]: @bradtopol  I agree with these edits (lines 15 and 144).  Looks good to me.  
( ꈍᴗꈍ)

Comment (wiggitywhitney) [reply content/en/docs/concepts/cluster-administration/logging.md:15]: /cc @bradtopol

**Comment by wiggitywhitney** [comment]
@bradtopol  Looks good to me.
( ꈍᴗꈍ)

**Comment by jimangel** [comment]
@nwanati can you address the conflicts? If you need any help, feel free to reach us on slack ([slack.k8s.io](https://slack.k8s.io/)) on the [SIG-Docs channel](https://kubernetes.slack.com/messages/sig-docs). Thanks!

**Pull request merged**